src/plugins/pcr/src/InSilicoPcrTask.cpp

```diff
@@ -49,9 +49,9 @@ bool InSilicoPcrProduct::isValid() const {
     return ta != Primer::INVALID_TM;
 }
 
-InSilicoPcrTask::InSilicoPcrTask(const InSilicoPcrTaskSettings &_settings)
-    : Task(tr("In Silico PCR"), TaskFlags(TaskFlag_ReportingIsSupported) | TaskFlag_ReportingIsEnabled | TaskFlags_FOSE_COSC),
-      settings(_settings), forwardSearch(nullptr), reverseSearch(nullptr), minProductSize(0) {
+InSilicoPcrTask::InSilicoPcrTask(const InSilicoPcrTaskSettings &settings)
+    : Task(tr("In Silico PCR"), TaskFlags(TaskFlag_ReportingIsSupported) | TaskFlag_ReportingIsEnabled | TaskFlag_FailOnSubtaskError),
+      settings(settings), forwardSearch(nullptr), reverseSearch(nullptr), minProductSize(0) {
```

Comment:
keep the '_' variant of the name: do not have 2 same-name variables in the same scope